.rubocop_todo.yml

```diff
@@ -0,0 +1,33 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2019-10-08 20:34:01 +0300 using RuboCop version 0.75.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Performance/RegexpMatch:
+  Exclude:
+    - 'lib/yavdb/sources/snyk_io.rb'
+
+# Offense count: 1
+# Configuration parameters: Max.
+RSpec/ExampleLength:
+  Exclude:
+    - 'spec/crawler_spec.rb'
+
+# Offense count: 1
+# Configuration parameters: CustomTransform, IgnoreMethods.
+RSpec/FilePath:
+  Exclude:
+    - 'spec/snyk_io_spec.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: Strict, EnforcedStyle.
+# SupportedStyles: inflected, explicit
+RSpec/PredicateMatcher:
```

Comment:
This one should be an easy fix, right? Can you try fixing it?

Reply:
kinda
I tried to change it to other matcher, but it's hard to run tests locally because it downloads all snyk repo :D and I'm unable to check if fix is correct

Reply:
You can check in CircleCI, only the last step should fail (pushing the results of coverage), or you can just run the smaller ones instead of all.

Reply:
okay, single test works fast enough for me
did you tried to use something like VCR to mock responses from services? it should speed up things
I see one issue with it that layout or something else can be changed on pages and tests will not catch it, but it can be solved by using integration tests for each db